ntoskrnl/io/pnpmgr/plugplay.c

```diff
@@ -136,7 +135,17 @@ IopGetDeviceObjectFromDeviceInstance(PUNICODE_STRING DeviceInstance)
             return NULL;
```

Comment:
Could this be simplified with:
```c
if (IopRootDeviceNode->PhysicalDeviceObject)
    ObReferenceObject(IopRootDeviceNode->PhysicalDeviceObject);
    
return IopRootDeviceNode->PhysicalDeviceObject;
```

Reply:
Yes, that would be valid. Though I don't consider it so much simpler that I feel a need to touch it.